app/views/admin/users/index.html.slim

```diff
@@ -1,15 +1,20 @@
 - provide(:title, 'Admin options')
 .container.content-area
-  h1 Admin options
+  h1 Interviewers
+  br
   .center
     .btn-group.btn-group-justified aria-label="..." role="group"
```

Comment:
@DimaSmagulov As for me, we need those buttons, but in header, as a dropdown. Because without them we need every time to go to dashboard, when we need to switch between questions and templates for example.